<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateViews</class>
 <widget class="QWidget" name="UpdateViews">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="4">
    <widget class="Classifier_List" name="gb_Old">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>300</height>
      </size>
     </property>
     <property name="title">
      <string>Old classifier list</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2"/>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="Classifier_List" name="gb_new">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>New classfier list</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3"/>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QToolButton" name="tb_SaveJson">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/img/res/outline_save_black_24dp.png</normaloff>:/img/res/outline_save_black_24dp.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="tb_OpenCsv">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/img/res/filter.png</normaloff>:/img/res/filter.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="tb_OpenJson">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/img/res/round_folder_open_black_24dp.png</normaloff>:/img/res/round_folder_open_black_24dp.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QGroupBox" name="gb_Sql">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Updated SQL</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Classifier_List</class>
   <extends>QGroupBox</extends>
   <header location="global">widgets/classifier_list.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
